hibernate-envers/src/test/java/org/hibernate/envers/test/integration/query/RevisionConstraintQuery.java

```diff
@@ -126,6 +126,7 @@ public void testRevisionsGtWithPropertyQuery() {
 				.addProjection( AuditEntity.revisionNumber() )
 				.add( AuditEntity.revisionNumber().gt( 1 ) )
 				.add( AuditEntity.property( "number" ).lt( 10 ) )
+				.addOrder( AuditEntity.revisionNumber().asc() )
```

Comment:
I am personally not a fan of adding an `#addOrder` here.  The method name defines we're testing two cases, the greater-than and property value check.  

Instead I would suggest we alter the assertion check like we do in other methods where we compare the contents of two sets as follows:
<pre>
Assert.assertEquals( TestTools.makeSet( 3, 4 ), new HashSet( result ) );
</pre>

Reply:
That's fine with me. I'll make the change and update the PR.